test/pummel/test-net-timeout.js

```diff
@@ -79,7 +79,7 @@ process.on('exit', function() {
   assert.ok(starttime != null);
   assert.ok(timeouttime != null);
 
-  diff = timeouttime - starttime;
+  var diff = timeouttime - starttime;
```

Comment:
ah yeah. forgot about that slight nuance. :P 

Reason they're not run with all the others is because they have the potential to run a long time, or overload the systems resources.